docs/openapi_spec.yaml

```diff
@@ -118,14 +133,17 @@ components:
         introSelfAssessment: Intermediate
         devices: [IPhone, Mac]
         skills:
-          - { title: iOS, description: Development of applications for the iOS mobile operating system, skillLevel: Novice}
-          - { title: CI/CD, description: Continuous integration and delivery/deployment using automated tools, skillLevel: Intermediate}
+          - { skillId: 3fa85f64-5717-4562-b3fc-2c963f66afa6, skillLevel: Novice, skillLevelRationale: Only dabbled a tiny bit with Swift}
+          - { skillId: 3fa85f64-6717-5562-c3fc-2c963f66afa6, skillLevel: Intermediate, skillLevelRationale: Wrote a project for a seminar that used this skill}
```

Comment:
Can we have more readable skillIDs? I think these are not good. Rather have just a string like `swift`, `uiux`, or `kubernetes`